docs/usage/tables/avro.md

```diff
@@ -0,0 +1,57 @@
+# Avro
+
+This page explains how to work with Avro tables in Unity Catalog.
+
+## Create Avro table
+
+Use the `bin/uc table create` command with the `--format AVRO` option to create a new Avro table in Unity Catalog.
+
+Run the following command with the correct `$DIRECTORY$` to create a new Avro table with 2 colummns: `some_numbers` and `some_letters`:
+
+```sh
+bin/uc table create \
+  --full_name unity.default.test \
+  --columns "some_numbers INT, some_letters STRING" \
+  --storage_location $DIRECTORY$ \
+  --format AVRO
+```
+
+After you run the `table create` command, your output should look something like this:
```

Comment:
Rephrase:

> Once executed, the output should be as follows:

Reply:
This seems like semantics to me. Could you give me some more context here / do we have a UC style guide that's been agreed upon that I should follow?

Reply:
Not really, but thought being concise makes sense.